.stylelintrc

```diff
@@ -0,0 +1,86 @@
+{
+  "extends": "stylelint-config-standard",
+  "plugins": [
+    "stylelint-order"
+  ],
+  "rules": {
+    "order/order": [
+      "custom-properties",
+      "declarations"
+    ],
+    "order/properties-order": [
+      "position",
```

Comment:
Just curious why you need to list a bunch of properties here 😂?

Reply:
I get lazy to manage the properties order, so I copied from this page. ><
ref: https://dotblogs.com.tw/explooosion/2018/09/30/141005

I think I can modify it in a smarter and more readable way later ;)

Reply:
Cool!